web/server.js

```diff
@@ -1,34 +1,210 @@
 // modules =================================================
-var express        = require("express");
-var app            = express();
-var bodyParser     = require("body-parser");
+var express		= require("express");
+var app			= express();
+var mongoose	   = require("mongoose");
+var bodyParser	 = require("body-parser");
 var methodOverride = require("method-override");
+var session = require("express-session");
```

Comment:
Mmmm, look at all those requires for OAuth. lol. It is definitely a beast on a whole different caliber!!